src/app/shared/services/skin/skin.service.spec.ts

```diff
@@ -32,6 +32,8 @@ const MOCK_SKIN_2: IAppSkin = {
 
 const MOCK_APP_CONFIG: Partial<IAppConfig> = {
   APP_HEADER_DEFAULTS: {
+    template: null,
+    show: true,
```

Comment:
Should this be 
```suggestion
    hidden: false,
```
?